manuscript/chapter2.md

```diff
@@ -1778,33 +1779,29 @@ const smallColumn = {
 };
 ~~~~~~~~
 
-After that you would use them in your columns: `<span style={smallColumn}>`.
-
-In general, you will find different opinions and solutions for style in React. You used pure CSS and inline style now. That's sufficient to get started.
-
-I don't want to be opinionated here, but I want to leave you some more options. You can read about them and apply them on your own. But if you are new to React, I would recommend to stick to pure CSS and inline style for now.
+After that, we use them in the columns: `<span style={smallColumn}>`.  There different opinions and solutions about style in React, but the pure inline CSS we used is sufficient for this tutorial.
 
-* [styled-components](https://github.com/styled-components/styled-components)
+* [Styled-components](https://github.com/styled-components/styled-components)
```

Comment:
styled small in this case, because styled-components is a library which hasn't a capital letter